### PR TITLE
Adding the new XDCR advanced filtering example

### PR DESCRIPTION
**Pull request by deniswsrosa**
Now the XDCR part of the tutorial is using the new advanced filtering instead of pure regex

### CODE REVIEW

**Comment by deniswsrosa** [comment]
Can you guys please review this PR?

File: modules/profile-store/pages/dotnet.adoc [hunk 1]
```diff
@@ -529,7 +529,9 @@ image:00503-remote-cluster.png[Remote Cluster]
 
 image:00504-add-rep.png[Add replication]
 
-Note that we have checked the option *Enable advanced filtering*. It will allow you to filter which documents should be replicated by applying a regex on the document id. In the case above, only documents with ids starting with *"user::"* will be replicated. As in our User Profile store, only the *User* has this prefix, it will be the only document type which will be replicated to other clusters. You can read more about it link:https://docs.couchbase.com/server/6.0/learn/clusters-and-availability/xdcr-filtering.html[here].
+Note that we have checked the option *Enable advanced filtering*. It will allow you to filter which documents should be replicated. Since Couchbase 6.5 you can filter documents using multiple attributes of your document using a syntax very similar to N1QL. You can read more about it link:https://docs-staging.couchbase.com/server/6.5/xdcr-reference/xdcr-filtering-expressions.html[here]
```
Comment (mgroves) [inline modules/profile-store/pages/dotnet.adoc]: @deniswsrosa I don't think you meant to link to docs-staging

File: modules/profile-store/pages/java.adoc [hunk 2]
```diff
@@ -549,7 +549,10 @@ image:00503-remote-cluster.png[Remote Cluster]
 
 image:00504-add-rep.png[Add replication]
 
-Note that we have checked the option *Enable advanced filtering*. It will allow you to filter which documents should be replicated by applying a regex on the document id. In the case above, only documents with ids starting with *"user::"* will be replicated. As in our User Profile store, only the *UserEntity* has this prefix, it will be the only document type which will be replicated to other clusters. You can read more about it link:https://docs.couchbase.com/server/6.0/learn/clusters-and-availability/xdcr-filtering.html[here].
+Note that we have checked the option *Enable advanced filtering*. It will allow you to filter which documents should be replicated. Since Couchbase 6.5 you can filter documents using multiple attributes of your document using a syntax very similar to N1QL. You can read more about it link:https://docs-staging.couchbase.com/server/6.5/xdcr-reference/xdcr-filtering-expressions.html[here]
```
Comment (mgroves) [inline modules/profile-store/pages/java.adoc]: another docs-staging link

File: modules/profile-store/pages/dotnet.adoc [hunk 3]
```diff
@@ -529,7 +529,9 @@ image:00503-remote-cluster.png[Remote Cluster]
...
+
+In the case above, only documents with the attribute *_class* equals to *com.cb.demo.userProfile.model.UserEntity* will be replicated. As in our User Profile store, only the *UserEntity* has this prefix, it will be the only document type which will be replicated to other clusters.
```
Comment (mgroves) [inline modules/profile-store/pages/dotnet.adoc]: Are you sure this applies to the dotnet tutorial? `_class` looks Spring specific

**Comment by mgroves** [comment]
@deniswsrosa there's a few comments for you

**Pull request merged**